### PR TITLE
Add event tests

### LINKED ISSUE #115: Add more Event related tests

### PR DESCRIPTION
**Pull request by SelwynAng**
Closes #115.

Added more events-related test methods and testcases.

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -125,17 +125,20 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
      * Executes the given {@code command}, confirms that <br>
      * - a {@code CommandException} is thrown <br>
      * - the CommandException message matches {@code expectedMessage} <br>
-     * - the address book, filtered member list and selected member in {@code actualModel} remain unchanged
+     * - the address book, filtered member list, filtered event list, selected member and selected event
```
Comment (wr1159) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:128]: Do we keep this as address book and replace address book with CCACommander in the later PR?

**Review by ph-nathan — APPROVED** [review]
LGTM

**Review by wr1159 — APPROVED** [review]
LGTM, just one comment about whether we should change the CCACommander now or later but I am leaning towards later once we wrap up 1.2

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java (hunk 1 above)
Comment (SelwynAng) [reply src/test/java/seedu/address/logic/commands/CommandTestUtil.java:128]: I think that changing it to CCACommander can be done in v1.3. We should open that as a new issue for v1.3.

**Pull request merged**